sdk/tables/azure-data-tables/azure/data/tables/aio/_table_client_async.py

```diff
@@ -569,3 +572,54 @@ async def upsert_entity(
             return _trim_service_metadata(metadata)
         except HttpResponseError as error:
             _process_table_error(error)
+
+    @distributed_trace
+    def create_batch(
+        self, **kwargs: Dict[str, Any]
+    ) -> TableBatchOperations:
+        """Update/Merge or Insert entity into table.
+
+        return: Table batch operation for inserting new operations
+        rtype: ~azure.data.tables.TableBatchOperations
+        :raises: None
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/async_samples/sample_batching_async.py
+                :start-after: [START batching]
+                :end-before: [END batching]
+                :language: python
+                :dedent: 8
+                :caption: Creating and adding an entity to a Table
+        """
+        return TableBatchOperations(
+            self._client,
+            self._client._serialize,  # pylint:disable=protected-access
+            self._client._deserialize,  # pylint:disable=protected-access
+            self._client._config,  # pylint:disable=protected-access
+            self.table_name,
+            self,
+            **kwargs
+        )
+
+    @distributed_trace_async
+    async def send_batch(
+        self,
+        batch: TableBatchOperations,
+        **kwargs: Dict[Any, str]
+    ) -> None:
+        """Commit a TableBatchOperations to send requests to the server
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/async_samples/sample_batching_async.py
+                :start-after: [START batching]
+                :end-before: [END batching]
+                :language: python
+                :dedent: 8
+                :caption: Using batches to send multiple requests at once
+        return: Table batch operation for inserting new operations
+        rtype: ~azure.data.tables.TableBatchOperations
```

Comment:
The return/rtype/raises tags should be above the admonition and are missing the first ":".
Also I think `rtype` should be TableTransactionResult and `raises` should be BatchErrorException